spec/features/agents/sectorisation/sectorisation_setup_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe "Agent can setup sectorisation", type: :feature do
+  let(:territory) { create(:territory, departement_number: "26") }
+  let(:ccas_valentinois) { create(:organisation, territory: territory, name: "CCAS Valentinois") }
+  let(:mds_drome) { create(:organisation, territory: territory, name: "MDS Drôme") }
+  let(:agent) { create(:agent, role_in_territories: [territory], admin_role_in_organisations: [ccas_valentinois, mds_drome]) }
+
+  before { login_as(agent, scope: :agent) }
+
+  it "works" do
+    visit admin_territory_sectorization_path(territory)
+    find("h2", text: "Secteurs").find(:xpath, "..").click
+    sleep 1
+    click_on "Créer un nouveau secteur"
+    fill_in :sector_name, with: "Secteur Nord"
+    fill_in :sector_human_id, with: "nord"
+    click_on "Enregistrer"
+    click_on "Ajouter une commune ou une rue"
+    sleep 1
+    find("#zone_city_name").native["value"] = "Valence"
```

Comment:
c’est la manière que j’ai trouvé de remplir un champ readonly sur une spec capybara sans js 😅  cf https://github.com/teamcapybara/capybara/issues/1178

j’ai essayé `find().set "Valence"` mais ça ne fonctionnait pas 🤷 malgré ce qui semble indiqué sur https://stackoverflow.com/questions/10804897/how-to-fill-hidden-field-with-capybara

Reply:
bien vu, ça pourrait mériter un commentaire ou (mieux) une extraction dans une méthode de helper